README.md

```diff
@@ -8,13 +8,24 @@ This repository servers two purposes.
    are deployed to each environment.
 2. It is the home of system-wide integration tests.
 
-To run the tests locally, use Python 3.6:
+Before you can run the tests, you need to setup your environment:
 
     pip install -r requirements.txt
-    CI_COMMIT_REF_NAME=staging make
+    # ping in #dcp-ops or #dcp-ops-help if you need valid service account credentials
+    export GOOGLE_APPLICATION_CREDENTIALS={PATH_TO_VALID_SERVICE_ACCOUNT_CREDS_WITH_ACCESS_TO_DSS}
+
+Instructions for running tests locally in python 3.6:
 
 Be aware that you will be uploading bundles to the staging deployment of the DCP and running secondary analysis.
```

Comment:
Are these instructions specific to staging?

Reply:
made them more generic